File: platforms/python/examples/core/texture/set_sampler_options.py
from fragmentcolor import Renderer, SamplerOptions
renderer = Renderer()
# 1x1 RGBA (white) raw pixel bytes
pixels = [255, 255, 255, 255]

texture = renderer.create_texture(pixels, size=[1, 1])
opts = {"repeat_x": True, "repeat_y": True, "smooth": True, "compare": None}
texture.set_sampler_options(opts)

File: platforms/web/examples/core/texture/set_sampler_options.js
import { Renderer } from "fragmentcolor";
const renderer = new Renderer();
// 1x1 RGBA (white) raw pixel bytes
const pixels = new Uint8Array([255, 255, 255, 255]);
const texture = await renderer.createTexture(pixels, { size: [1, 1] });
const opts = { repeatX: true, repeatY: true, smooth: true, compare: null };
texture.setSamplerOptions(opts);